src/routes/admin/+page.svelte

```
@@ -11,67 +12,102 @@
+		updateInterval: 30,
+		theme: { ...THEME_DEFAULTS }
 	});
 
 	let runningTime = $state(0);
 	let selector = $state('en');
+	let logoUrlError = $state('');
+
+	const logoUrl = $derived(localConfig.theme.logoUrl || PUBLIC_OBA_LOGO_URL);
+	const regionName = $derived(localConfig.theme.regionName || PUBLIC_OBA_REGION_NAME);
+
+	const ALL_TOKENS = { ...SITE_TOKENS, ...BOARD_TOKENS };
```

ESLint

[error] 'ALL_TOKENS' is assigned a value but never used.

(no-unused-vars)